tox.ini

```diff
@@ -17,8 +17,8 @@ passenv =
     OS_*
 deps =
     .[dev]
-    {lint,func,func-debug}: zaza @ git+https://github.com/openstack-charmers/zaza.git#egg=zaza
-    {lint,func,func-debug}: zaza-openstack @ git+https://github.com/openstack-charmers/zaza-openstack-tests.git#egg=zaza.openstack
+    {lint,format-code,func,func-debug}: zaza @ git+https://github.com/openstack-charmers/zaza.git#egg=zaza
+    {lint,format-code,func,func-debug}: zaza-openstack @ git+https://github.com/openstack-charmers/zaza-openstack-tests.git#egg=zaza.openstack
```

Comment:
Are zaza and zaza-openstack needed on the `format-code` virtual env?

Reply:
Yes it's needed for `isort`.  I added `[testenv:func-debug] envdir = {toxworkdir}/func` and `[testenv:format-code] envdir = {toxworkdir}/lint` to avoid unnecessary installation.